tests/data/reference_graphs/quantized/hawq/mobilenet_v2_device_VPU__mode_liberal__ratio_2.5__setup_type_propagation_based.dot

```diff
@@ -1,273 +1,273 @@
+"40 MobileNetV2/Sequential[features]/InvertedResidual[3]/Sequential[conv]/NNCFConv2d[2]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]/symmetric_quantize" [color=purple, id=40, label="2_bit__WFQ_#40_G9", scope="MobileNetV2/Sequential[features]/InvertedResidual[3]/Sequential[conv]/NNCFConv2d[2]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]", style=filled, type=symmetric_quantize];
 "41 MobileNetV2/Sequential[features]/InvertedResidual[3]/Sequential[conv]/NNCFConv2d[2]/conv2d" [color=lightblue, id=41, label=conv2d, scope="MobileNetV2/Sequential[features]/InvertedResidual[3]/Sequential[conv]/NNCFConv2d[2]", style=filled, type=conv2d];
-"42 MobileNetV2/Sequential[features]/InvertedResidual[3]/Sequential[conv]/BatchNorm2d[3]/batch_norm" [id=42, label=batch_norm, scope="MobileNetV2/Sequential[features]/InvertedResidual[3]/Sequential[conv]/BatchNorm2d[3]", style=filled, type=batch_norm];
-"43 MobileNetV2/Sequential[features]/InvertedResidual[3]/Sequential[conv]/BatchNorm2d[3]/AsymmetricQuantizer/asymmetric_quantize" [color=red, id=43, label="4_bit__AFQ_#30", scope="MobileNetV2/Sequential[features]/InvertedResidual[3]/Sequential[conv]/BatchNorm2d[3]/AsymmetricQuantizer", style=filled, type=asymmetric_quantize];
-"44 MobileNetV2/Sequential[features]/InvertedResidual[3]/__add__" [id=44, label=__add__, scope="MobileNetV2/Sequential[features]/InvertedResidual[3]", style=filled, type=__add__];
-"45 MobileNetV2/Sequential[features]/InvertedResidual[3]/AsymmetricQuantizer/asymmetric_quantize" [color=red, id=45, label="4_bit__AFQ_#35", scope="MobileNetV2/Sequential[features]/InvertedResidual[3]/AsymmetricQuantizer", style=filled, type=asymmetric_quantize];
-"46 MobileNetV2/Sequential[features]/InvertedResidual[4]/Sequential[conv]/ConvBNReLU[0]/NNCFConv2d[0]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]/symmetric_quantize" [color=purple, id=46, label="2_bit__WFQ_#35", scope="MobileNetV2/Sequential[features]/InvertedResidual[4]/Sequential[conv]/ConvBNReLU[0]/NNCFConv2d[0]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]", style=filled, type=symmetric_quantize];
+"42 MobileNetV2/Sequential[features]/InvertedResidual[3]/Sequential[conv]/BatchNorm2d[3]/batch_norm" [id=42, label="batch_norm_#42", scope="MobileNetV2/Sequential[features]/InvertedResidual[3]/Sequential[conv]/BatchNorm2d[3]", style=filled, type=batch_norm];
+"43 MobileNetV2/Sequential[features]/InvertedResidual[3]/Sequential[conv]/BatchNorm2d[3]/AsymmetricQuantizer/asymmetric_quantize" [color=purple, id=43, label="2_bit__AFQ_#43_G44", scope="MobileNetV2/Sequential[features]/InvertedResidual[3]/Sequential[conv]/BatchNorm2d[3]/AsymmetricQuantizer", style=filled, type=asymmetric_quantize];
```

Comment:
That's not expected for VPU config. The minimum acceptable bitwidth for the add layer is 4, not 2.

![image](https://user-images.githubusercontent.com/4014476/103093856-ae22cc00-460c-11eb-86b0-5954fbc7d065.png)

Reply:
Fixed a bug in liberal bitwidth assignment and added a runtime check to validate final config against the initial proposed search space set of configs.